scripts/create_dev_db.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
We don't need this script. This command already takes care of everything:

```
node ../node_modules/db-migrate/bin/db-migrate up
```

The only thing we need is to make sure this command is automatically called every time we start the dev version of the server